ViewModel/AggregatedStageDamage/AggregatedStageDamageElement.cs

```diff
@@ -17,15 +17,17 @@ public class AggregatedStageDamageElement : ChildElement
         public bool CanEdit { get; }
         public int SelectedWSE { get; set; }
         public int SelectedStructures { get; set; }
-
         public List<StageDamageCurve> Curves { get; }
         public bool IsManual { get; }
+        public List<ImpactAreaFrequencyFunctionRowItem> ImpactAreaFrequencyRows { get; }
 
         #endregion
         #region Constructors
 
-        public AggregatedStageDamageElement(String name, string lastEditDate, string description,int selectedWSE, int selectedStructs, List<StageDamageCurve> curves, bool isManual, int id) : base(id)
```

Comment:
Rather than storing all the properties in the Stage damage element, can we just store the VM, like we did with Graphical? in the AnalyticalFlowFrequencyElement ?